ui/css/treeherder-job-buttons.css

```diff
@@ -51,19 +52,25 @@
 }
 
 .group-content {
-  margin-left: -3px;
+  margin: 0 0;
+  white-space: nowrap;
+  
 }
 
 .group-content::before {
   content: "(";
+  padding-left: 1px;
+
 }
 
 .group-content::after {
   content: ")";
+  margin-left: =-1;
```

Comment:
Sorry I've seen negative margins but have never seen this before.

Is `=-1` intentional? I am curious what it means.